source/documentation/using_the_api/api_reference.md

```diff
@@ -165,21 +176,26 @@ Example response:
 ]
 ```
 
-You can then download the latest item. For example, entry 265 in the above code snippet. Follow the [guidance for downloading items](#items).
-
-### `GET /records/{field-name}/{field-value}`
-
-Get all records that share a field-value for a particular field. For example, all local authorities marked as county councils from the `local-authority-eng` register. 
+### <a name="get-records-field-name-field-value">Get all records that share a `field-value` for a particular `field-name`</a>
 
-> Results from this API call are paginated. This call will return the first 100 records from the first page of the register. Use `page-size` to define the number of records you want and `page-index` to define the pages you want. The maximum `page-size` is 5000.
+Endpoint: `GET /records/{field-name}/{field-value}/`
 
-Example URL: `https://local-authority-eng.register.gov.uk/records/local-authority-type/CTY`
+Parameters: 
+* page-index (Optional): Collection page number. Defaults to 1.
+* page-size (Optional): Collection page size. Defaults to 100. Maximum is 5000. 
 
-Example request: `curl --request GET --url https://local-authority-eng.register.gov.uk/records/local-authority-type/CTY --header 'Accept: application/json' --header 'Authorization: YOUR-API-KEY-HERE'`
-
-Example response:
+```
+GET /records/{field-name}/{field-value}/ HTTP/2
+Host: local-authority-eng.register.gov.uk
+Accept: application/json
+Authorization: YOUR-API-KEY-HERE
+```
 
 ```
+HTTP/2 200
+Content-Type: application/json
+Link: <?page-index=1&page-size=3>
```

Comment:
This `Link` header needs probably a `rel=next` and then values should be higher. Also the `GET` part should contain the parameters, at least the `page-size` as it doesn't default to 100.